lib/engine/step/g_18_ms/draft_distribution.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
```

Comment:
Can any of the 1846 code be shared?

Reply:
I do not know if that would be good? 
The 18MS draft is a straightforward one, where each player buy one private - they cannot pass. It is open. Variants of this exists in 1893 and 1876.
Possibly 1848 and 18CZ could reuse this type of draft.

So in the same way as we have several different auction maybe we need to have several different drafts?

Reply:
Draft from 1846 is now shared, and moved up one level.